tests/meteor.sh

```diff
@@ -0,0 +1,18 @@
+#!/bin/sh
+
+set -ex
+
+npm install -g npm@latest
+
+export PUPPETEER_DOWNLOAD_PATH=~/.npm/chromium
+
+export METEOR_LOCAL_DIR=~/.meteor
+
+export MONGO_URL="mongodb://localhost:27017/"
+
+export MONGO_OPLOG_URL="mongodb://localhost:27017/local?replicaSet=rs0&directConnection=true"
+
+TINYTEST_FILTER="mongo-livedata - doc fetcher" ./packages/test-in-console/run.sh --once
+TINYTEST_FILTER="mongo-livedata - oplog - cursorSupported" ./packages/test-in-console/run.sh --once
+TINYTEST_FILTER="mongo-livedata - oplog - entry skipping" ./packages/test-in-console/run.sh --once
+TINYTEST_FILTER="mongo-livedata - oplog - x - implicit collection creation" ./packages/test-in-console/run.sh --once
```

Comment:
All these tiny tests get run in the same script so currently we do not have details on each specific sub-test's outcome. I will figure out a better way to pass them to the command runner in the future.